audio-clockworkpi-a06/PKGBUILD

```diff
@@ -0,0 +1,24 @@
+# Maintainer: lonewsk <lwsk -at- nodot.fr>
+
+pkgname=audio-clockworkpi-a06
+pkgver=1.0
+pkgrel=1
+pkgdesc="ClockworkPi DevTerm A06 Gearbox"
+arch=('aarch64')
+license=('LGPLv2')
+depends=('alsa-lib')
+groups=('clockworkpi-a06')
+source=("git://github.com/maxfierke/clockworkpi-a06-post-install")
```

Comment:
Please use the Manjaro ARM repos as the upstream for this. My repo is not necessarily up-to-date with them.

Reply:
Thanks will fix this shortly